snapper-rollback.py

```
@@ -138,6 +164,12 @@ def main():
         dev = config.get("root", "dev")
     except configparser.NoOptionError as e:
         dev = None
+        
+    try:
+        mountpoint_newsubvol = config.get("root", "mountpoint_newsubvol")
+    except configparser.NoOptionError:
+        mountpoint_newsubvol = None
+        LOG.info("mountpoint_newsubvol not configured, skipping pacman db.lck removal")
```

rather than doing a try/except here, I think it would be better to add a new `--sanitize` flag. If a failure occurs, the script fails fast and loud